src/dns/aardvark.rs

```diff
@@ -7,23 +7,33 @@ use std::fs;
 use std::fs::File;
 use std::fs::OpenOptions;
 use std::io::prelude::*;
+use std::io::{self, BufRead};
 use std::io::Result;
 use std::net::IpAddr;
+use std::net::Ipv6Addr;
 use std::net::Ipv4Addr;
 use std::path::Path;
 use std::process::{Command, Stdio};
 
 const SYSTEMD_CHECK_PATH: &str = "/run/systemd/system";
 const SYSTEMD_RUN: &str = "systemd-run";
 
-#[derive(Debug, Clone)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
Please do not add Serialize, Deserialize. These types should never be written as json